packages/apputils-extension/src/index.ts

```diff
@@ -160,6 +170,22 @@ const themes: JupyterFrontEndPlugin<IThemeManager> = {
       }
     });
 
+    commands.addCommand(CommandIDs.themeScrollbars, {
+      label: 'Theme Scrollbars',
+      isToggled: () => manager.themeScrollbars(currentTheme),
```

Comment:
This isn't getting toggled as I execute the command

Reply:
It seems to work for me, so I need more details. What exactly are you doing (ie how are you invoking the `execute` of the command? By clicking on the "Theme Scrollbars" menu item, or in a different way?), and why exactly does it seem broken (is the check mark not showing up/disappearing correctly in the menu? Or was it something else you noticed)?

Reply:
I fixed this. Aside from scrollbar theming being mostly broken in Firefox, the logic for the `isToggled` callback was slightly off.